configs/pg.env

```diff
@@ -1,17 +1,20 @@
 pgconf_max_connections=1000
-pgconf_shared_buffers=96GB
```

Comment:
We have to keep in mind that when we merge it, it affects all new non-3.3.0 deployments as well as existing ones that checkout the toolkit, and AWS.
Are these settings proven with AWS and 3.1.0 ?

Reply:
- aws has problems with this in general. you shouldn't set most of these in AWS to begin with as they have configs per machine setup that are actually more optimal.
- 3.1.0 has "only" a problem with the leader participation.

what i wonder however is why this is there and these settings are not just in the docker psql config that we generate when we create the docker image. now if someone runs without the compose something they don't hvae the proper settings? or alternatively we should maybe enforce the use of docker compose

Reply:
Because it may need to be overridden by the user, hence they are here. But also, some of it could go directly to our default stock config.

Reply:
yes it may be. i would however hope that (almost) all of it is in our default config too.

Reply:
I am not so sure about that. @jukoson could you maybe check what's in the box per default?

Reply:
What's the "box" ?

Reply:
I agree I think with both of you here - some of the config could and should go into our container by default, such as the parallel leader participation. If we include it in the container by default, there is no risk with version incompatibilities between s64-compose/product-image

Reply:
@jukoson that does not work when one updates the container and re-uses the data dir. In that case, the configuration does not get applied. I rather vote for checking the version number and making changes accordingly on container startup.

Reply:
As part of our docker entrypoints we eg. check whether the postgresql.conf has fpga information. If it doesn't we add it. If it does, we update it.
We could use the same mechanism to just set the leader participation flag. With that, we avoid this issue. There is only one issue when downgrading the S64 DA version - but I am not sure whether we really need to protect against this

Reply:
@sdressler Please check the comment again